src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dayRentalCost = 40;
+  const smallDiscount = 20;
+  const bigDiscount = 50;
+  const smallVocation = 3;
+  const bigVocation = 7;
```

Comment:
You can move them outside the function. And variables that are static called constants should be in UPPER_CASE